SU2_CFD/include/fluid/CCoolProp.hpp

```diff
@@ -0,0 +1,139 @@
+/*!
+ * \file CCoolProp.hpp
+ * \brief Defines the state-of-the-art fluid model from CoolProp library.
+ * \author P. Yan, G. Gori, A. Guardone
+ * \version 7.3.1 "Blackbird"
```

Comment:
Go over the new files and update to 7.4.0

Reply:
change done